src/java.base/share/classes/jdk/internal/foreign/abi/Binding.java

```diff
@@ -268,8 +269,21 @@ static BoxAddress boxAddress(long byteSize) {
         return new BoxAddress(byteSize, 1, true);
     }
 
-    static UnboxAddress unboxAddress() {
-        return UnboxAddress.INSTANCE;
+    // alias
+    static SegmentOffset unboxAddress() {
+        return segmentOffsetNoAllowHeap();
+    }
+
+    static SegmentBase segmentBase() {
+        return SegmentBase.INSTANCE;
+    }
+
+    static SegmentOffset segmentOffsetAllowHeap() {
```

Comment:
I suppose `heapSegmentOffset` is not good because this binding applies to both native and heap segments, right?

Reply:
Yes